ksqldb-common/src/main/java/io/confluent/ksql/util/QueryApplicationId.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import io.confluent.ksql.query.QueryId;
+
+/**
+ * Util to build query application ids.
+ */
+public final class QueryApplicationId {
```

Comment:
Class would benefit from a couple of tests!  Maybe one test for persistent and one for transient.

Reply:
ack